Dockerfile

```
@@ -40,6 +40,10 @@ RUN add-apt-repository -y ppa:git-core/ppa && \
     zip && \
   rm -rf /var/lib/apt/lists/*
 
+# revise system defaults
+RUN sed --in-place --regexp-extended 's/^( *)(alias l.*)/\1#\2/' /etc/skel/.bashrc && \
```

Ideally, the range should be restricted to only directory listing aliases (i.e. `l`, `la`, `ll`, `ls`, and etc.) rather than blindly altering all alias assignments starting with `l`;

```suggestion
RUN sed --in-place --regexp-extended 's/^( *)(alias l[als]?=*)/\1#\2/' /etc/skel/.bashrc && \
```

However, the currently shipped `.bashrc` contains no alias assignments starting with `l` other than the targets and unlikely to change unless with a newer release of the image and hence postponing the change shall be harmless.